components/Table/Table.stories.tsx

```diff
@@ -333,112 +433,161 @@ export const WithCaption: ComponentStory<any> = (args) => {
-          <TableForStory {...args}>
-            <VisuallyHidden asChild>
-              <Caption>{title}</Caption>
-            </VisuallyHidden>
-            <Thead>
-              <Tr>
-                <Th>Firstname</Th>
-                <Th>Lastname</Th>
-                <Th>Status</Th>
-                <Th>Role</Th>
-              </Tr>
-            </Thead>
-            <Tbody>
-              <Tr>
-                <Td>John</Td>
-                <Td>Doe</Td>
-                <Td>
-                  <Badge variant="green">Connected</Badge>
-                </Td>
-                <Td>Developer</Td>
-              </Tr>
-              <Tr>
-                <Td>Johny</Td>
-                <Td>Depp</Td>
-                <Td>
-                  <Badge variant="orange">AFK</Badge>
-                </Td>
-                <Td>Actor</Td>
-              </Tr>
-              <Tr>
-                <Td>Natalie</Td>
-                <Td>Portman</Td>
-                <Td>
-                  <Badge variant="green">Connected</Badge>
-                </Td>
-                <Td>Actor</Td>
-              </Tr>
-              <Tr>
-                <Td>Luke</Td>
-                <Td>Skywalker</Td>
-                <Td>
-                  <Badge variant="red">Disconnected</Badge>
-                </Td>
-                <Td>Star wars</Td>
-              </Tr>
-            </Tbody>
-          </TableForStory>
-        </div>
-        <div>
-          <Heading id={id} size="4" as="h1">
-            {title}
-          </Heading>
-          <TableForStory aria-describedby={id} {...args}>
-            <Thead>
-              <Tr>
-                <Th>Firstname</Th>
-                <Th>Lastname</Th>
-                <Th>Status</Th>
-                <Th>Role</Th>
-              </Tr>
-            </Thead>
-            <Tbody>
-              <Tr>
-                <Td>John</Td>
-                <Td>Doe</Td>
-                <Td>
-                  <Badge variant="green">Connected</Badge>
-                </Td>
-                <Td>Developer</Td>
-              </Tr>
-              <Tr>
-                <Td>Johny</Td>
-                <Td>Depp</Td>
-                <Td>
-                  <Badge variant="orange">AFK</Badge>
-                </Td>
-                <Td>Actor</Td>
-              </Tr>
-              <Tr>
-                <Td>Natalie</Td>
-                <Td>Portman</Td>
-                <Td>
-                  <Badge variant="green">Connected</Badge>
-                </Td>
-                <Td>Actor</Td>
-              </Tr>
-              <Tr>
-                <Td>Luke</Td>
-                <Td>Skywalker</Td>
-                <Td>
-                  <Badge variant="red">Disconnected</Badge>
-                </Td>
-                <Td>Star wars</Td>
-              </Tr>
-            </Tbody>
-          </TableForStory>
-        </div>
-      </Flex>
-    </>
+      </div>
+    </Flex>
   );
 };
 
+export const Empty: ComponentStory<any> = (args) => (
```

Comment:
Would you mind adding one more scenario?
- Interactive without `Tfoot`

https://user-images.githubusercontent.com/38889364/158206465-cf046777-7bac-47ff-a3c2-8d878a5ef487.mov

Reply:
@paulocfjunior this one was tricky :sweat_smile: 

I had to set `overflow: 'hidden'` on `Table`, because it is not possible to reliably apply radii on latest row when there's no `Tfoot`: reverse sibling order is not yet possible in css.

Please review and give me your opinion

Reply:
Indeed it seems tricky, we seem to be limited by the technology on this 😛 
Your implem looks good to me 👍